Data/Sys/GameSettings/RMK.ini

```diff
@@ -19,3 +19,6 @@ EmulationIssues =
 
 [Video_Hacks]
 EFBEmulateFormatChanges = True
+
+[Video_Settings]
+MSAA = 0 # Fix shadow transparency
```

Comment:
I don't thing it is a good idea to disable antialiasing because it's results depends on the graphic drivers and the devs do various changes sometimes to antialiasing that might affect it. Would you be here in case something changes and track it to change it back? Also this also disables SSAA, are you sure both msaa and ssaa have an issue?